### PR TITLE
CBR share member SDK

### PR DESCRIPTION
**Pull request by tischrei**

### CODE REVIEW

**Comment by tischrei** [comment]
If someone has a better idea to implement the proxy method "add_members" in backup module or better in member module can leave me a hint. If not, this is also ok :-D

File: otcextensions/sdk/cbr/v3/_proxy.py [hunk 1]
```diff
@@ -385,3 +386,94 @@ def dissociate_resources(self, vault, resources):
...
+             instance.
+        """
+        backup = self._get_resource(_backup.Backup, backup)
+        vault = self._get_resource(_vault.Vault, vault)
+        return self._update(
+            _member.Member, member, backup_id=backup.id,
+            status=status, vault_id=vault.id)
+
+    def delete_member(self, backup, member, ignore_missing=True):
+        """Delete a single CBR vault.
```
Comment (lego963) [inline otcextensions/sdk/cbr/v3/_proxy.py]: Update description please :)

File: otcextensions/tests/unit/sdk/cbr/v3/test_member.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
```
Comment (lego963) [inline otcextensions/tests/unit/sdk/cbr/v3/test_member.py:1]: You forgot to add 
```sh
#!/usr/bin/env python3
```

**Review by lego963 — CHANGES_REQUESTED** [review]

File: otcextensions/tests/unit/sdk/cbr/v3/test_member.py (hunk 2 above)
Comment (tischrei) [reply otcextensions/tests/unit/sdk/cbr/v3/test_member.py:1]: all of our test  files do not have the shebang line registered. So I don't see a (real) issue here.

File: otcextensions/sdk/cbr/v3/_proxy.py (hunk 1 above)
Comment (tischrei) [reply otcextensions/sdk/cbr/v3/_proxy.py]: fixed

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by lego963 — DISMISSED** [review]

**Comment by tischrei** [comment]
Can anyone tell me why eco/check fails in Zuul. When I test it via tox it runs through...

**Comment by lego963** [comment]
> Can anyone tell me why eco/check fails in Zuul. When I test it via tox it runs through...

sorry for delay 
```
otcextensions.tests.unit.osclient.load_balancer.v1.test_pool_member.TestListPoolMember.test_list_default [0.002305s] ... FAILED
```

UPD: I've started `re-run`

**Comment by tischrei** [comment]
> > Can anyone tell me why eco/check fails in Zuul. When I test it via tox it runs through...
> 
> sorry for delay
> 
> ```
> otcextensions.tests.unit.osclient.load_balancer.v1.test_pool_member.TestListPoolMember.test_list_default [0.002305s] ... FAILED
> ```
> 
> UPD: I've started `re-run`

And you get this error as well when you run it  by yourself?

EDIT: Dependency issue in tox env. So I didn't saw the issue by myself. Now, it is there. Will fix it.

**Review by lego963 — APPROVED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Pull request merged**